testdata/p4_16_samples/free-form-annotation.p4

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
you need to also commit the result files produced by running this test.
They should be in in p4_16_samples_outputs.

Reply:
Done in afadac0.